src/Symfony/Component/Console/Application.php

```diff
@@ -706,6 +706,19 @@ public function renderException(\Exception $e, OutputInterface $output)
     {
         $output->writeln('', OutputInterface::VERBOSITY_QUIET);
 
+        $this->doRenderException($e, $output);
+
+        if (null !== $this->runningCommand) {
+            $output->writeln(sprintf('<info>%s</info>', sprintf($this->runningCommand->getSynopsis(), $this->getName())), OutputInterface::VERBOSITY_QUIET);
+            $output->writeln('', OutputInterface::VERBOSITY_QUIET);
+        }
+    }
+
+    /**
+     * @internal
```

Comment:
I think I'd suggest remove this one: using things cross components/bundles is a trigger that this is not so internal anymore - cross-versions constraints would anyway prevent doing non BC things here